### PR TITLE
Header menu and contact section

### PR DESCRIPTION
**Pull request by herokudev**
### Features for this branch:

- A Navigation bar.
- Three content sections:
        1. 	Books list.
        2. 	Add book form.
        3. 	Contact info.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,6 +49,23 @@ <h2 class="form-title center">
...
+          <ul class="contact-details">
+            <li>Central America Base:</li>
+            <li>Our email: mail@mail.com.gt</li>
+            <li>Our Phone: +502 6578124</li>
+            <li>Our Address: Nameless Street, Numberless House, Guatemala</li>
+          </ul>
+        </div>
+    </div>
+    <footer class="border border-2 border-dark mt-auto px-1">
+        <h2 class="fs-6 fw-bold">Copyright &copy; <script>document.write(new Date().getFullYear())</script> All Rights Reserved</h2>
```
Comment (rloterh) [inline index.html:65]: - [ ] Your `copyright` centered and footer does not stay at the bottom. We actually want our our design to stay to as much as possible follow the design of the [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_full_website.png), and so therefore kindly keep the `copyright`  to the left and the footer should at all times stay  the bottom

Your design:
image

**The wireframe design:**
image

File: index.html [hunk 2]
```diff
@@ -7,10 +7,21 @@
...
-<body>
-    <div class="container mt-4">
+<body>    
+    <nav class="d-flex justify-content-between border border-2 border-dark p-1 w-100">
+        <a id="nav-home" href="#" class="text-decoration-none text-dark fw-bold fs-6">Awesome Books</a>
+        <ul class="d-flex list-unstyled m-0 align-items-center">
+          <li><a id="nav-list" class="links text-decoration-none text-dark fs-6 fw-bold px-2" href="#">List</a></li>
+          <li><a id="nav-add" class="links text-decoration-none text-dark fs-6 fw-bold px-2" href="#">Add new</a></li>
+          <li><a id="nav-contact" class="text-decoration-none text-dark fs-6 fw-bold px-1" href="#">Contact</a></li>
+        </ul>
```
Comment (rloterh) [inline index.html:20]: - [ ] The right menu elements in your navbar is missing the dividers between each menu element, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.
- [ ] To enhance the user experience of this design kindly ensure that an active menu link is highlighted(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted.

**Your design menu:**
image

**The wireframe menu design:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Kingsley & , Herbert 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 


## Required Changes ♻️
- [ ] _Kindly check the comments under the review._


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZawadiSumba66 — APPROVED** [review]
Hi Team 👋 , 
## STATUS: APPROVED 👍 
Great job designing as recommended by the previous reviewer. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

### NOTE:
Kindly ensure that you always update your README file to contain the current version of the app.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KingsleyMcSimon** [comment]
Thanks a whole lot, it was really an experience working on the project.

**Pull request merged**